cli/src/commit_templater.rs

```diff
@@ -212,6 +218,36 @@ impl<'repo> IntoTemplateProperty<'repo, Commit> for CommitTemplatePropertyKind<'
     }
 }
 
+/// Function that translates method call node of self type `T`.
+// The lifetime parameter 'repo could be replaced with for<'repo> to keep the
+// method table away from a certain lifetime. That's technically more correct,
+// but I couldn't find an easy way to expand that to the core template methods,
+// which are defined for L: TemplateLanguage<'repo>. That's why the build fn
+// table is bound to a named lifetime, and therefore can't be cached statically.
```

Comment:
I suppose a somewhat positive consequence is that the customization can decide to add the custom template methods only for repos that support it, so `jj log -T custom_keyword` might fail in regular Git repos (assuming the custom keyword is specific to a custom backend). Hmm, but that would mean that default templates get a bit harder to define because they can't be in shared config. So I suppose the customization will want to always define the same custom keywords anyway and instead make them return some placeholder values when used with the wrong backend. I'm just thinking out loud; no action required.

Reply:
> I suppose the customization will want to always define the same custom keywords anyway and instead make them return some placeholder values when used with the wrong backend.

I agree. Even if it returns a "unsupported backend" error, it's better than random "no such method" error.